photon-ml/src/main/scala/com/linkedin/photon/ml/algorithm/FactoredRandomEffectCoordinate.scala

```diff
@@ -28,22 +27,29 @@ import com.linkedin.photon.ml.util.VectorUtils
 import org.apache.spark.rdd.RDD
 
 /**
-  * The optimization problem coordinate for a factored random effect model
+  * The optimization problem coordinate for a random effect model using matrix factorization
```

Comment:
@joshvfleming 
The Coordinate files look like they've been substantially changed on GitHub, but mostly it was moving the order of functions around to be consistent between classes. This may be easier to see by checking out the PR locally.